.github/workflows/rails-integration-tests.yml

```diff
@@ -20,10 +22,14 @@ jobs:
       - name: Check User Permission
         if: steps.checkAccess.outputs.require-result == 'false'
         run: |
-          echo "${{ github.triggering_actor }} does not have permissions on this repo."
-          echo "Current permission level is ${{ steps.checkAccess.outputs.user-permission }}"
-          echo "Job originally triggered by ${{ github.actor }}"
+          echo "${ trigger-actor } does not have permissions on this repo."
+          echo "Current permission level is ${ permission }"
+          echo "Job originally triggered by ${ actor-name }"
           exit 1
+        env:
+          trigger-actor: ${{ github.triggering_actor }}
+          actor-name: ${{ github.actor }}
+          permission: ${{ steps.checkAccess.outputs.user-permission }}
```

Comment:
Nitpick: Could we put the `env` stuff before the `run` - just removes a bit of misdirection